src/Provider/Provider.Web/ViewModels/Part2/Skills/SkillsViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using Esfa.Recruit.Shared.Web.ViewModels;
+using Esfa.Recruit.Shared.Web.ViewModels.Skills;
+
+namespace Esfa.Recruit.Provider.Web.ViewModels.Part2.Skills
+{
+    public class SkillsViewModel : SkillsEditModel, ISkillsViewModel
```

Comment:
We have been trying to avoid this. ViewModels and EditModels are two separate concerns that don't go together and ViewModels should not inherit from EditModels. Otherwise we might as well use one class for input and output and use [BindNever] attributes for ignoring parts of request binding.

Reply:
This is just cut and pasted from exiting code. Most of our View Models inherit from Edit Models.

Reply:
I've removed the inheritance between the View and Edit model